meson.build

```diff
@@ -5,7 +5,7 @@ project(
 	default_options: [
 		'c_std=c11',
 		'warning_level=2',
-		'werror=true',
+		'werror=false',
```

Comment:
We probably want to keep this to `true`.

Reply:
Hmmm.. yes, that was not meant to slip in...